### PR TITLE
feat!: precompile css-modules, update spacing

### PR DESCRIPTION
**Pull request by jquense**
no build step needed.

### CODE REVIEW

File: src/Layout.module.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+);
+
+@each $content in $justify-content {
+  .justify-#{$content} {
+    justify-content: $content;
+  }
+}
+
+@each $level, $spacer in $spacers {
+  .pad-#{$level} {
```
Comment (taion) [inline src/Layout.module.css]: i'm somewhat confused by this approach with levels here

it seems like – if we're using CSS variables anyway, why not just pass down the padding as a variable of some sort, esp given that we're just calculating the padding as `level * 4`

likewise for things like the align, justify bits...

can't we just do e.g.

```
justify-content: var(--fc-layout-justify-content)
```

and not have to define a bunch of extra classes?

Comment (taion) [reply src/Layout.module.css]: or does that lead to undesirable cascade things?

Comment (jquense) [reply src/Layout.module.css]: Ooo that's a great idea.

Comment (jquense) [reply src/Layout.module.css]: O I guess the concern there is all `as` components would need to pass through style as well as className. Idk if that's a reasonable assumption

**Comment by jquense** [comment]
k here is as minimal as i'm clever enough to do i think

**Comment by jquense** [comment]
I really wish flexbox supported `gap` like css grid...

**Comment by jquense** [comment]
WDT?

File: www/src/pages/index.js [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-  html {
-    box-sizing: border-box;
-  }
-  *,
-  *::before,
-  *::after {
-    box-sizing: inherit;
-  }
-`;
+// const styles = css`
```
Comment (taion) [inline www/src/pages/index.js]: ?

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './Layout.css';
+import { defaultProps, propTypes } from './PropTypes';
+
+export const Spacing = {
```
Comment (taion) [inline src/index.js]: what is this for?

**Review by taion — APPROVED** [review]
i like this approach

**Pull request merged**